### DISCUSSION TITLE
Issue #3217 open but already solved

### DISCUSSION THREAD

**Discussion by ljwoods2** [MDAnalysis/mdanalysis#4400] [Development]
Unless I'm mistaken, it looks like this issue (https://github.com/MDAnalysis/mdanalysis/issues/3217) is already solved- the class transformations.boxdimensions.set_dimensions allows dimensions for multiple timesteps already. Here's some code I ran to verify, let me know if I'm missing something!

```python
from MDAnalysis.tests.datafiles import PSF, DCD
import MDAnalysis as mda
from MDAnalysis.transformations import set_dimensions
import numpy as np

u = mda.Universe(PSF, DCD)
dim = []
for i in range(98):
    coord = i % 4
    dim.append([coord, coord, coord, 90, 90, 90])

transform = mda.transformations.boxdimensions.set_dimensions(dim)
u.trajectory.add_transformations(transform)
for ts in u.trajectory:
    print(ts)
```

**Comment by orbeckst** [comment]
Yes, you're correct. The issue can be closed.

Will do so.
